### ISSUE TITLE
V2 escape characters doesn't work

### ISSUE THREAD

**Issue by daedalus28** [date-fns/date-fns#968]
Escaping characters in `format` isn't working in V2. Here's may sample code - I tried to set up a runkit but runkit didn't seem to work with `@next`:

```js
let {format}  = require('date-fns')
console.log(format(new Date(), '[Q]Q'))
```

Assuming we're in the fourth quarter (which we are until Jan 1 2019):

Output in V2: `[4]4`
Expected Output (and output in V1): `Q4`

**Comment by kossnocorp** [comment]
@Pau1fitz what docs? `format` doc contains two mentioning of that.

**Comment by Pau1fitz** [comment]
You are right, I see it now when I have gone to the latest version in the top right. I guess I meant [here]( https://date-fns.org/v1.30.1/docs/format), but now I think of it, maybe a note in the setting up guide to say that for version 2 of the docs make sure you select the correct version from the dropdown above. Also, in [v2.0.0-alpha.1 ](https://date-fns.org/v2.0.0-alpha.1/docs/format) square brackets are still used, so when I changed it to v2, although not the latest one, it was not saying to use a single quote. 

Anyway, as you said it is in the docs already so maybe it is clear enough, although it looks as if some people are missing it so maybe an idea to be considered.
